crates/autopilot/src/run_loop.rs

```diff
@@ -148,48 +175,50 @@ impl RunLoop {
         }
     }
 
+    async fn update_caches(&self, prev_block: &mut Option<H256>, store_events: bool) -> BlockInfo {
```

Comment:
Extracted from next_auction() so we can just run this read-only path in a loop in the standby instance.